firebase-crashlytics/src/main/java/com/google/firebase/crashlytics/internal/CrashlyticsWorker.java

```diff
@@ -79,6 +85,7 @@ public <T> Task<T> submit(Callable<T> callable) {
    * <p>Returns a <code>Task</code> which will be resolved with null upon successful completion of
    * the runnable, or throws an <code>ExecutionException</code> if the runnable throws an exception.
    */
+  @CanIgnoreReturnValue
```

Comment:
Also here to mentioning here `Submit a blocking runnable will block the execution `